crates/uv-pep440/src/version.rs

```diff
@@ -1416,6 +1418,39 @@ impl FromStr for VersionPattern {
     }
 }
 
+/// Lifetime and indexing workaround to allow accessing the release as `&[u64]` even though the
+/// digits may be stored in a compressed representation.
```

Comment:
Since this is part of the public API, I'd suggest keeping the first sentence to one line, and then adding more (if necessary) in a paragraph below it. Otherwise, this will show up as an overlong summary in the type listing.